torchvision/prototype/transforms/_transform.py

```diff
@@ -114,24 +114,12 @@ def _extract_params_for_v1_transform(self) -> Dict[str, Any]:
         # Overwrite this method on the v2 transform class if the above is not sufficient. For example, this might happen
         # if the v2 transform introduced new parameters that are not support by the v1 transform.
         common_attrs = nn.Module().__dict__.keys()
-        params = {
+        return {
             attr: value
             for attr, value in self.__dict__.items()
             if not attr.startswith("_") and attr not in common_attrs
         }
 
-        # transforms v2 has a more complex handling for the `fill` parameter than v1. By default, the input is parsed
```

Comment:
We no longer need this, since the original `fill` is kept on the transform and the internal `_fill` is not picked up since it is private.

Reply:
sweet! We can remove the comment that refers to `fill` above as well then